falcon/testing/client.py

```diff
@@ -808,6 +814,11 @@ async def _simulate_request_asgi(
             overrides `body` and sets the Content-Type header to
             ``'application/json'``, overriding any value specified by either
             the `content_type` or `headers` arguments.
+        msgpack(Msgpack serializable): A Msgpack document to serialize as the
```

Comment:
It is not documented what happens in the case both `json` **and** `msgpack` are specified.

Reply:
Hello @vytas7, sorry for taking so long, I added comments that say that the content-type will be MEDIA-MSGPACK when we pass both json and msgpack as parameters.